src/daemon/instance_settings_handler.cpp

```diff
@@ -153,6 +154,48 @@ void update_disk(const QString& key, const QString& val, mp::VirtualMachine& ins
     }
 }
 
+bool to_bool(const QString& key, const QString& val)
```

Comment:
This is implemented in `BoolSettingSpec`. Instead of copy+pasting from there, why don't you instantiate a helper object and use that?

Reply:
Hey @ricab, thanks for your quick review!

I thought in avoid copy+pasting, but I opted for not a `QString` and comparing against it. Just true or false. Do you still think it's OK to reuse the existing implementation?

Reply:
I see what you mean: you want to return a `bool` and compare against that in the code, right? But you can still achieve that with an extra step. For example, just something like this to replace the contents of `to_bool`: `return BoolSettingSpec{key, "false"}.interpret(val) == "true";`

Reply:
I wanted to avoid that string comparison, and throw directly instead of returning a value before throwing. But well, it is true that the code looks much better the way you propose. Since time is not critical here, I changed it. Thanks!